### DISCUSSION TITLE
iOS simulator isn't respecting tiling.

### DISCUSSION THREAD

**Discussion by sadiksaifi** [nikitabobko/AeroSpace#797] [questions-and-answers]
**Describe the bug**
iOS simulator isn't respecting tiling. Whenever the iOS simulator opens, it just always sticks to the left and overlaps other windows. The window underneath that makes a weird sort of space at the right. I am attaching video of that behavior.

https://github.com/user-attachments/assets/4944a5ae-c04d-4be0-9410-ac399143ba7f

**Related issues you could find**
Please make sure to search for duplicates.
If you couldn't find the duplicates, linking related issues is helpful anyway.

**Comment by nikitabobko** [comment]
You have `accordion` layout active, not `tiles` layout active
